docs/Home.md

```diff
@@ -0,0 +1,108 @@
+
+        https://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY 
+KIND, either express or implied. See the License for the specific language governing 
+permissions and limitations under the License.
+-->
+# Google Compute Engine Plugin
+The Google Compute Engine (GCE) Plugin allows you to use GCE virtual machines (VMs) with Jenkins to execute build tasks. GCE VMs provision quickly, are destroyed by Jenkins when idle, and offer Preemptible VMs that run at a much lower price than regular VMs.
+
+
+View Google Compute Engine on the plugin site for more information.
+
+## Usage
+### IAM Credentials
+1. Create a service account using the Google Cloud SDK.
+
+   ```
+    gcloud iam service-accounts create jenkins-gce
+   ```
+
+2. Add the instanceAdmin and serviceAccountUser roles to the service account.
+
+   ```
+   export PROJECT=$(gcloud info --format='value(config.project)') 
+   export SA_EMAIL=$(gcloud iam service-accounts list --filter="name:jenkins-gce"
+    --format='value(email)') 
+   gcloud projects add-iam-policy-binding --member serviceAccount:$SA_EMAIL --role
+   roles/compute.instanceAdmin --role roles/iam.serviceAccountUser $PROJECT
+   ```
+
+3. Download a JSON Service Account key for your newly created service account. Take note
+   of where the file was created, you will upload it to Jenkins in a subsequent step.
+
+   ```
+   gcloud iam service-accounts keys create --iam-account $SA_EMAIL jenkins-gce.json
+   ```
+4. In Jenkins, click the Credentials button on the left side of the screen. Then click 
+   System.
+5. Click Global credentials then **Add credentials** on the left.
+6. In the Kind dropdown, select Google Service Account from private key.
+7. Enter your project name then select your JSON key that was created in the preceding
+   steps.
+8. Click OK.
+
+### Google Compute Engine configuration
+Each GCE configuration can point to a different GCP project. Follow the steps below to create one.
+
+ 1. Go to Manage Jenkins, then Configure System
+ 2. At the bottom of the page there will be a button labeled Add a new cloud, click the 
+    button then click Google Compute Engine.
+ 3. Enter a name for your cloud configuration and the Project ID that you will be using
+    to deploy the instances.
+ 4. In the Service Account Credentials dropdown, select the credentials that you uploaded
+    earlier.
+
+### Instance configurations
+An instance configuration allows you to map an instance to a set of labels that Jenkins
+ can use to determine when a job requires a particular type of instance. You can create 
+ many instance configurations per GCE configuration.
+
+1. Click on the Add button.
+2. In the Name Prefix field, choose a prefix for the name of the instances that will be
+    deployed.
```

Comment:
There's one extra space in the second lines for 2, 4, and 6 which I think is causing it to break the line in the view.

Reply:
Yikes, didn't show up in my preview in my text editor. Done